lib/Controller/Stats.php

```diff
@@ -710,7 +710,7 @@ public function export(Request $request, Response $response)
         $toDt = $sanitizedParams->getDate('toDt');
         $displayId = $sanitizedParams->getInt('displayId');
         $tempFileName = $this->getConfig()->getSetting('LIBRARY_LOCATION') . 'temp/stats_' . Random::generateString();
-        $dateFormat = $sanitizedParams->getString('dateFormat'); // UTC/CMS local time
+        $dateFormat = $sanitizedParams->getCheckbox('dateFormat');
```

Comment:
Again, I think having this parameter called `dateFormat` is confusing - it isn't a date format. I think I suggested an alternative parameter name in my last review?

Reply:
I have done that in the latest commit.